plugins/outputs/iotdb/iotdb_test.go

```diff
@@ -0,0 +1,377 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package iotdb
+
+import (
+	"fmt"
+	"math"
+	"testing"
+	"time"
+
+	"github.com/apache/iotdb-client-go/client"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+var (
```

Comment:
Why do you need them? They are the default aren't they?

Reply:
oh, you are right! I will remove these meaningless variables. 
But I'd like to remain `testPort` as a const like this:
```
const testPort = "6667"
```
This is because other functions using container have to specify listening ports. Reusing a const string is better than using "6667" everywhere, I think.